firmware/src/src/targets/nrf52/analog.cpp

```diff
@@ -4,16 +4,16 @@
 #include <string.h>
 #include <zephyr.h>
 
+#include "log.h"
 #include "serial.h"
 
-
 // Simple analog input method
 // this just reads a sample then waits then returns it
 
 // ADC Sampling Settings
 // doc says that impedance of 800K == 40usec sample time
 
-#define ADC_DEVICE_NAME DT_LABEL(DT_ALIAS(adcctrl))
+//#define ADC_DEVICE_NAME DT_LABEL(DT_ALIAS(adcctrl))
```

Comment:
what do you want to do here?  change this define or just keep impl inline as below?

Reply:
Removed the define